packages/common/src/type-helpers.ts

```diff
@@ -12,12 +12,11 @@ export function checkExtends<_Orig, _Copy extends _Orig>(): void {
 
 export type Replace<Base, New> = Omit<Base, keyof New> & New;
 
-export type MakeOptional<Base, Keys extends keyof Base> = Omit<Base, Keys> & Partial<Pick<Base, Keys>>;
```

Comment:
removed, as it was unused. alternatively could add ignore comment if we want to have for future use

Reply:
👍